sonar-project.properties

```diff
@@ -0,0 +1,19 @@
+sonar.projectKey=redhat-appstudio_jvm-build-service
+sonar.organization=redhat-appstudio-1
+
+# This is the name and version displayed in the SonarCloud UI.
+sonar.projectName=jvm-build-service
+
+
+# Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
+sonar.sources=.
+
+# Path to sources
+sonar.sources=.
+sonar.exclusions=**/*test*, **/vendor/**, **/zz_generated.*, **/*.yaml, **/java-components/**
+
+# Source encoding
+sonar.sourceEncoding=UTF-8
+
+# Exclusions for copy-paste detection
+sonar.cpd.exclusions=**/*test*
```

Comment:
Is this only for the GoLang scanning? Can this be a hidden file or specified directly in the workflow?

Reply:
How come you renamed it to a hidden file in other repositories e.g. https://github.com/redhat-appstudio/e2e-tests/blob/main/.sonarcloud.properties , https://github.com/redhat-appstudio/service-provider-integration-operator/blob/main/.sonarcloud.properties ?